src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,13 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
         // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(5);
```

Comment:
It's a good practice to shut down the ExecutorService using try-finally block or try-with-resources to ensure that the executor is properly shut down in case of any exceptions that might occur during the execution of the tasks.